src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -112,6 +112,11 @@ class GemUtils {
 
                 systemProperties 'file.encoding' : 'utf-8'
             }
+            project.javaexec {
```

Comment:
Yes, we can. The install process have proven itself, so we can towards some tweaks. I do not want to lose the ability to run out of process, as the isolation it provides means is great. However, I think it would relatively easy to have an oprion to run in-process. The experimental work I have done for embedding Rake can refactored so that appropriate code ends up in base.

The only thing to remember is that the in-process install with run (1) under exactly the same JDK version, Gradle starts with an d (2) will use jruby.version as the jruby version.

(1) is not really an issue as people rarely override the JDKversion when using JRubyExec
(2) will probably not be a major issue for most use-cases

<div>-------- Original message --------</div><div>From: "R. Tyler Croy" <notifications@github.com> </div><div>Date:15/04/2015  17:16  (GMT+00:00) </div><div>To: jruby-gradle/jruby-gradle-plugin <jruby-gradle-plugin@noreply.github.com> </div><div>Subject: Re: [jruby-gradle-plugin] generate .jrubydir files for installed gems
  (#108) </div><div>
</div>In src/main/groovy/com/github/jrubygradle/GemUtils.groovy:

> @@ -112,6 +112,11 @@ class GemUtils {
>  
>                  systemProperties 'file.encoding' : 'utf-8'
>              }
> +            project.javaexec {
Is there no way to accomplish this without spinning up another JVM :/

—
Reply to this email directly or view it on GitHub.